server/parsec/events.py

```diff
@@ -146,6 +146,25 @@ def dump_as_apiv4_sse_payload(self) -> bytes:
         )
 
 
+class EventChatReceived(BaseModel, ClientBroadcastableEvent):
+    model_config = ConfigDict(arbitrary_types_allowed=True, strict=True)
+    type: Literal["CHATRECEIVED"] = "CHATRECEIVED"
```

Comment:
```suggestion
    type: Literal["CHAT_MESSAGE_RECEIVED"] = "CHAT_MESSAGE_RECEIVED"
```

Should be the same value as the `discriminant_value` used for the variant `ChatMessageReceived` defined in `events_listen.json5`